libbeat/common/transport/tlscommon/types.go

```diff
@@ -127,13 +127,15 @@ const (
 	VerifyFull TLSVerificationMode = iota
 	VerifyNone
 	VerifyCertificate
+	VerifyLegacyCommonName
 )
 
 var tlsVerificationModes = map[string]TLSVerificationMode{
-	"":            VerifyFull,
+	"":            VerifyLegacyCommonName,
```

Comment:
This changes the default value from `full` to `common_name`, which should also be reflected in the <beat>.reference.yml files (they are currently still showing `full` as default). 

With changing the default to `common_name` you ensure that the more restrict behavior of go 1.15 is shielded from the users and they actually see the same default behavior as they have now, right?

Reply:
> With changing the default to common_name you ensure that the more restrict behavior of go 1.15 is shielded from the users and they actually see the same default behavior as they have now, right?

Exactly.

Reply:
The change is not fully bc for users that have configured the verification mode. The setting is supposed to default to "full". How about changing "full" and "" to include the check for Common Name, but introduce a "strict" verification mode that relies in the stdlib?